### PR TITLE
Features/#412 store busmaps used for clustering

### PR DESCRIPTION
**Pull request by CarlosEpia**
Solve #412  and #410

### CODE REVIEW

**Comment by CarlosEpia** [comment]
This PR has been modified in the way that it does not depend on csv files.

File: etrago/appl.py [hunk 1]
```diff
@@ -1,496 +0,0 @@
...
-    print(datetime.datetime.now())
-    etrago.session.close()
-    # plots
-    # make a line loading plot
-    # plot_line_loading(network)
-    # plot stacked sum of nominal power for each generator type and timestep
-    # plot_stacked_gen(network, resolution="MW")
-    # plot to show extendable storages
-    # storage_distribution(network)
-    # extension_overlay_network(network)
```
Comment (ClaraBuettner) [inline etrago/appl.py:496]: This looks a bit strange. Did you accidentally deleted the appl.py?

**Comment by ClaraBuettner** [comment]
One general comment: It looks like there are many changes caused by using `black`. In eTraGo, we used to use another tool/style called `pep8`. We could also agree on using `black`, but I would like to discus that first. If we want to change it, it would make sense to apply black in separate commits, otherwise it is a bit challenging to understand what was changed in the commit :)

**Comment by CarlosEpia** [comment]
I considered it easier and more comprehensible to create a new branch and PR #431 without black and the first implementation which used the csv files.

**Pull request closed without merging**